### DISCUSSION TITLE
Food Effect Buffs

### DISCUSSION THREAD

**Discussion by DrEnzyme** [space-wizards/space-station-14#29334] [Ideaguys]
At the moment this is a very early sketch of a concept I would like to implement to give food a bit more use. The idea is to give food various small buffs that make it worth eating. Some ideas:

* Soups and Spicy Food: Raise your body temperature and heal a little bit of cold damage over time.
* Burgers and Fatty foods: Reduce the effects of alcohol/space drugs and very slowly heal poison damage.
* Hearty foods: Increase your body's natural healing rate 
* Caffinated drinks: slight movement speed boost.

I'm not sure exactly how you'd go about the implementation. Some of this stuff makes sense to be a reagent (like caffeine), other stuff might make more sense to be its own separate "food buff" that you only get for eating (not grinding) a meal, and it would make sense that you can only have one or two of these food buffs at a time to avoid stacking them.

I'm looking for general feedback, implementation ideas and other proposals that have come up for cooking that I can use ideas from. Something like this is a bit above my ability when it comes to modifying the codebase so it would also be a "several projects down the line" sort of thing.

**Comment by ShadowCommander** [comment]
We do want to add buffs to food eventually, so it's good to start getting it written down, but you'll probably get a bit more engagement on Discord or the forums https://forum.spacestation14.com/c/development/feature-requests/52

**Comment by DrEnzyme** [comment]
Thanks Shadow I'll post there.
